lib/msal-common/src/response/SPAResponseHandler.ts

```diff
@@ -26,7 +26,7 @@ import { ServerError } from "../error/ServerError";
 /**
  * Class that handles response parsing.
  */
-export class ResponseHandler {
+export class SPAResponseHandler {
```

Comment:
These changes seem unrelated to the `NodeStorage` changes, could this be split into two PRs?

Reply:
Yes. I am retaining this as the PRs following up are dependent on this change, but will adhere to this norm in creating future PRs.